comp/dogstatsd/server/server.go

```diff
@@ -438,7 +438,7 @@ func (s *server) start(context.Context) error {
 		}
 	}
 
-	s.health = health.RegisterLiveness("dogstatsd-main")
```

Comment:
Rather than having to update a lot of code, consider using [functional options](https://github.com/tmrts/go-patterns/blob/master/idiom/functional-options.md) to pass additional flags. In addition to fewer places to update, `RegisterLiveness(..., health.RunOnce)` is more explicit than `RegisterLiveness(..., true)` and will easier to extend later.

Reply:
Thanks for the suggestion

I wanted to do this at first but I thought it would make the code a bit harder to understand (I mean when reading the health package, while it will definitely be more understandable for external components). 

I updated the code with your suggestion, I agree it is cleaner this way